src/daos/build/install-single-host-dgw.sh

```diff
@@ -202,6 +242,8 @@ function build_ceph()
 }
 
 # Defaults
+CCACHE_BUILD_PATH='/opt/ccache-build'
+CCACHE_PATH='/opt/ccache'
```

Comment:
I understand `ccache-build`, but where is `ccache` being used?

Reply:
For SSC hosts, its imperative that the default folder is not used because its shared between all hosts.  I ran into issues building on multiple hosts.  The script updates the ccache.conf to utilize the path.  If you read the confluence page, this is described.